Failure on x86_64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.34706&attempt_id=d989af59-4b97-434d-8455-94fff4bc4878)

<details><summary>Partial log (click to expand)</summary><p>

```
fi # Make sure the highscore file has non-zero length to make rpmlint happy...
mkdir: cannot create directory '/var': Permission denied
/nix/store/yq03c2ny43mc24j7dq5riznzb09ddhpq-bash-4.4-p12/bin/bash: line 2: /var/xjump/highscores: No such file or directory
make[2]: *** [Makefile:1010: install-exec-hook] Error 1
make[2]: Leaving directory '/build/source'
make[1]: *** [Makefile:936: install-exec-am] Error 2
make[1]: Leaving directory '/build/source'
make: *** [Makefile:881: install-am] Error 2
builder for '/nix/store/h55j8jh7riqcsiaadp9kadfkayb23iy2-xjump-2.9.3.drv' failed with exit code 2
error: build of '/nix/store/h55j8jh7riqcsiaadp9kadfkayb23iy2-xjump-2.9.3.drv' failed
```
</p></details>

--- a/pkgs/games/xjump/default.nix
+++ b/pkgs/games/xjump/default.nix
@@ -16,6 +16,7 @@ stdenv.mkDerivation rec {
 
   meta = with stdenv.lib; {
     description = "The falling tower game";
+    license = licenses.gpl2;
     maintainers = with maintainers; [ pmeunier ];
   };
 }

